concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,221 @@
+
+If we determine that an issue you report requires an embargo, we will discuss 
+this with you and try to find a reasonable expiry date (aka “unembargo date”), 
+as well as who should be included in the list of need-to-know people.
+
+In any case, embargoes are a process that should be used sparingly, as they 
+are painful and time-consuming.
+
+For more information about embargoes and vulnerability disclosure, you can 
+refer to [this article](https://aliceevebob.com/2018/01/09/meltdown-and-spectre-thinking-about-embargoes-and-disclosures).
+
+Lastly, in no way should an embargo be understood as a means to avoid making 
+Enarx faults public. Transparency is paramount to the Enarx project.
+
+--- 
+### Enarx Security Advisories
+This page lists all security advisories, vulnerabilities and fixes in Enarx 
+itself.
+
+For security issues, vulnerabilities and fixes in underlying TEEs, both in 
+SDKs and in hardware, please refer to our 
+[page on the subject](https://github.com/enarx/enarx/wiki/TEEs-vulnerabilities-and-attacks/).
+
+If you think you have found a security issue with Enarx, we encourage you to 
+report it.
+To do so, you may follow the process described in the 
+[Vulnerability Disclosure and Embargo Policy](https://github.com/enarx/enarx/wiki/Vulnerability-disclosure-and-embargo-policy) page.
+
+#### List of Enarx Security Advisories
+N/A
+
+--- 
+
+## Reference
+
+Much of this RFC is inspired by the [Rust Security Policy](https://www.rust-lang.org/policies/security),
+as well as [RFPolicy 2.0](https://dl.packetstormsecurity.net/papers/general/rfpolicy-2.0.txt), 
+which it refers to.
+
+## Drawbacks
+
+- The process as currently described makes use of email and OpenPGP. PGP 
+  email is notoriously hard to get right, even more so over time.
+
+## Rationale and alternatives
+
+- The most basic alternative is not to have a policy and to simply wait for
+people to disclose their findings. This does nothing to provide guarantees 
+to researchers and users that their work and time will be respected and to 
+encourage further disclosures.
+
+## Prior art
+
+Rust process mentioned above.
+
+## Unresolved questions
+As mentioned, encrypted email is not a good solution. It’s really hard to get 
+PGP encrypted email right, it offers no Perfect Forward Secrecy (PFS) and key 
+management is horrible (cf. [this analysis](https://latacora.micro.blog/2019/07/16/the-pgp-problem.html), 
+for instance). However, in the case of just offering a way for people outside 
+Enarx to send sensitive information with _some_ security, it may be 
+acceptable. 
+Key management is still hard: how many copies of the private key? where do 
+they live? how do we revocate if maintainers leave project? etc.
+
+In light of this:
+- Can we *not* offer email as a way communication channel while remaining 
+universal enough to allow everyone to report security issues in Enarx?
+- Should we treat email as a public channel, similar to our chat?
```

Comment:
There's a distinction, but we should be careful.

Reply:
A distinction in nature or in risk?
At the end of the day, i believe more and more strongly that we should treat email as a public channel, at least in terms of anything that might need confidentiality. It may not be entirely technically true, but it fosters the right attitude towards the levels of privacy and confidentiality one can expect from email overall.

Reply:
Agreed.